syft/serde.py

```diff
@@ -157,6 +155,74 @@ def deserialize(
     return _detail(worker, simple_objects)
 
 
+def serialize_tensor(tensor) -> bin:
```

Comment:
This is clever

Reply:
I think we might want to make this and deserialize tensor private functions as the user should not be calling them